sources/mysql/toddl_test.go

```diff
@@ -25,6 +25,103 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestToSpannerTypeInternal(t *testing.T) {
+
+	_, errCheck := toSpannerTypeInternal(schema.Type{"bool", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
```

Comment:
This error message needs to be corrected.

Reply:
Same for all tests

Reply:
done